lib/UnoCore/Targets/Android/app/src/main/AndroidManifest.xml

```diff
@@ -15,7 +47,11 @@
                  android:description="@string/description"
                  android:largeHeap="@(Project.Android.LargeHeap:IsSet:Test(@(Project.Android.LargeHeap:Bool),false))"
                  android:allowBackup="@(Project.Android.AllowBackup:IsSet:Test(@(Project.Android.AllowBackup:Bool),false))"
-                 android:usesCleartextTraffic="@(Project.Android.UsesCleartextTraffic:IsSet:Test(@(Project.Android.UsesCleartextTraffic:Bool),false))">
+                 android:usesCleartextTraffic="@(Project.Android.UsesCleartextTraffic:IsSet:Test(@(Project.Android.UsesCleartextTraffic:Bool),false))"
+#if @(Project.Android.Bundle.CompressNativeLibraries:Test(1, 0))
+                 android:extractNativeLibs="false"
+#endif
```

Comment:
I feel this project property should've been named `Android.ExtractNativeLibs`, similar as in AndroidManifest, maybe to make it more familar for Android users.

The whole `#if..#endif` block can probably be replaced with `android:extractNativeLibs="@(Project.Android.ExtractNativeLibs:IsSet:Test(@(Project.Android.ExtractNativeLibs:Bool),true))"`

Reply:
ok, done